core/src/main/scala/cats/syntax/either.scala

```diff
@@ -306,6 +308,14 @@ final class RightOps[A, B](val right: Right[A, B]) extends AnyVal {
   def leftCast[C]: Either[C, B] = right.asInstanceOf[Either[C, B]]
 }
 
+final class EitherIdOps[A](val obj: A) extends AnyVal {
```

Comment:
Ah, don't know how I missed that. I will submit a PR  to fix OptionIdOps as soon as I get back to my laptop. Thanks for pointing it out.  I still don't think being consistent with previous mistakes provides  much value here.